graphql/resolver.go

```diff
@@ -25,7 +25,8 @@ func New(apiURL string) Config {
 			sc: &data.DBConnector{URL: apiURL},
 		},
 	}
-	c.Directives.RequireSuperUser = func(ctx context.Context, obj interface{}, next graphql.Resolver) (interface{}, error) {
```

Comment:
Have we considered adding some documentation or warnings when a new user who is not yet an admin anywhere is added as an admin to a project? Because the user adding them probably doesn't realize that they are simultaneously giving this user the power to create new evergreen projects.

Reply:
I don't think we should necessarily notify them every time but I think it would make sense to add something here 
<img width="742" alt="image" src="https://user-images.githubusercontent.com/26491602/226479302-50555244-67e1-4db3-bb6d-da18a75e0911.png">


and it would also be added to our docs